server/api_server/sync_endpoint.py

```
@@ -47,40 +47,102 @@ def handle_sync_get():
+        f"content_length={request.content_length}",
+        f"remote_addr={request.remote_addr}"
+    ])
+
+    # ---- RAW BODY (critical for debugging encoding / encryption issues)
+    try:
+        raw = request.get_data(cache=False)
+        mylog("verbose", [
+            f"[SYNC API] raw_bytes_len={len(raw)}",
+            f"[SYNC API] raw_preview={raw[:200]}"
+        ])
+    except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)